### PR TITLE
New repo README

### PR DESCRIPTION
**Pull request by omri374**
New README.
Contains:
1. The goals of the project (why are we doing this?)
2. Main features focused on V2
3. Links to selected docs
4. Changed the How to contact us section

Link to MD file: https://github.com/microsoft/presidio/blob/feature/new_readme/README.MD

### CODE REVIEW

**Comment by navalev** [comment]
can we remove the "how it works" gif and just leave a link to the demo site?

File: README.MD [hunk 1]
```diff
@@ -4,124 +4,64 @@
...
-## How to contact us?
+1. [Using pip](docs/installation.md#using-pip)
+2. [Using Docker](docs/installation.md#using-docker)
 
-If you have a usage question, found a bug or have a suggestion for improvement, please file a [Github issue](https://github.com/microsoft/presidio/issues).
-For other matters, please email presidio@microsoft.com
+## Running Presidio
 
+1. [Running Presidio via code](docs/deployment-samples/python/index.md)
+2. [Running Presidio via HTTP](docs/deployment-samples/docker/index.md)
```
Comment (navalev) [inline README.MD:51]: Running Presidio as an HTTP service

Comment (omri374) [reply README.MD]: fixed

**Review by navalev — APPROVED** [review]

**Comment by omri374** [comment]
@cobypeled could you please review?

**Comment by omri374** [comment]
@navalev replaced image with demo gif

**Review by navalev — APPROVED** [review]

**Review by cobypeled — APPROVED** [review]

**Pull request merged**